packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -2242,8 +2242,12 @@ export interface IChoiceGroupOptionStyleProps {
     // (undocumented)
     hasImage?: boolean;
     // (undocumented)
+    imageHeight?: number;
+    // (undocumented)
     imageIsLarge?: boolean;
     // (undocumented)
+    imageWidth?: number;

```

Comment:
Ideally, these types would be `number | string` to match `Image#width` and `Image#height`:

https://github.com/OfficeDev/office-ui-fabric-react/blob/34ca27b108ea853976d061c32e80c31156eaf846/packages/office-ui-fabric-react/src/components/Image/Image.types.ts#L231

But we redefine the type of `imageSize` here:

https://github.com/OfficeDev/office-ui-fabric-react/blob/34ca27b108ea853976d061c32e80c31156eaf846/packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts#L129

So that can be deferred.. just pointing out a pre-existing inconsistency